README.md

```diff
@@ -136,6 +136,7 @@
 - [Tank Royale](https://tankroyale.io/) - Cartoony fast-paced top-down multiplayer shooter.
 - [TANX](https://tanx.io) - Online multiplayer tank battle game.
 - [Venge](https://venge.io/) - 3D Multiplayer FPS shooter.
+- [Aritelia](https://aritelia.io/) - Open world social MMO with sandbox building and collectable NFTs.
```

Comment:
Just wanted to check that you didn't want this in the NFT section?

Reply:
We struggled with this, not sure :)

Initially NFTs were going to be a smaller part of the experience, but gradually they are becoming quite part of the game. And given the NFT category on this list can get some boost from this, agreed, let's move this there.

Let me update the PR!